.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] As requested by the previous reviewer [here](https://github.com/cheronojudith/module1-first-project/pull/3#:~:text=Nice%20work%20creating,and%20test.md), Kindly make sure test.md is also ignored as node_module :recycle: 

__Kindly make sure to use the same PR for review, as it makes it easier to track the project's evolution :recycle:__

Reply:
Thank you @GedeonTS for correcting me have ignored the test.md